back/app/services/print_custom_fields_service.rb

```diff
@@ -0,0 +1,222 @@
+# frozen_string_literal: true
+require 'prawn'
+require 'prawn/measurement_extensions'
+
+class PrintCustomFieldsService
+  attr_reader :custom_fields, :params, :previous_cursor
+
+  QUESTION_TYPES = %w[select multiselect text text_multiloc multiline_text html_multiloc linear_scale]
+  FORBIDDEN_HTML_TAGS_REGEX = /<\/?(div|span|ul|ol|li|em|img|a){1}[^>]*\/?>/
+
+  def initialize(custom_fields, params)
+    @custom_fields = custom_fields
+    @params = params
+    @previous_cursor = nil
+  end
+
+  def create_pdf
+    pdf = Prawn::Document.new(page_size: 'A4')
+
+    if params[:name] == 'true' then
+      render_text_field_with_name(
+        pdf,
+        I18n.with_locale(locale) { I18n.t('form_builder.pdf_export.full_name') }
+      )
+    end
+
+    if params[:email] == 'true' then
+      render_text_field_with_name(
+        pdf,
+        I18n.with_locale(locale) { I18n.t('form_builder.pdf_export.email_address') }
+      )
+    end
+
+    custom_fields.each_with_index do |custom_field, i|
+      # First custom_field should always be a page.
```

Comment:
This comment is not corrrect anymore, since with the idea form the first field will be a 'section' instead. But since we ignore sections for now functionally it doesn't matter